server/config.py

```diff
@@ -1,3 +1,8 @@
 import os
 
-TEAM_NAME = os.environ['TEAM_NAME']
+# TEAM_NAME = os.environ['TEAM_NAME']
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+SQLALCHEMY_ECHO = False
+SQLALCHEMY_TRACK_MODIFICATIONS = True
+SQLALCHEMY_DATABASE_URI = "postgresql://localhost/rosebud"
```

Comment:
ideally make the value of this config variable an environment variable (like TEAM_NAME example above)